src/queries/db_functions.js

```diff
@@ -1,169 +0,0 @@
-const dbConnection = require('../database/db_connection.js');
-
-
-const getData = (cohort, cb) => {
-
-  dbConnection.query(`SELECT weeks.num , weeks.week_title , mentors.githubuser from weeks inner join cohort_mentor on weeks.num = cohort_mentor.week_num inner join mentors on mentors.githubuser = cohort_mentor.mentor_user  where  cohort_mentor.cohort_id  = ${cohort}`, (err, res) => {
-    if (err) {
-      cb(err);
-    } else {
-      cb(null, res.rows);
-    }
-  });
-};
-
-const weeksMentors = (cohort, cb) => {
-  var weeks = {
-    1: {
-      mentors: []
-    },
-    2: {
-      mentors: []
-    },
-    3: {
-      mentors: []
-    },
-    4: {
-      mentors: []
-    },
-    5: {
-      mentors: []
-    },
-    6: {
-      mentors: []
-    },
-    7: {
-      mentors: []
-    },
-    8: {
-      mentors: []
-    },
-    10: {
-      mentors: []
-    },
-    11: {
-      mentors: []
-    },
-    12: {
-      mentors: []
-    },
-    13: {
-      mentors: []
-    },
-    14: {
-      mentors: []
-    },
-    15: {
-      mentors: []
-    },
-    16: {
-      mentors: []
-    },
-  }
-  getData(cohort, (err, res) => {
-    if (err) {
-      console.log(err);
-    } else {
-
-      res.map((object, i) => {
-        weeks[object.num].mentors.push(object.githubuser);
-        weeks[object.num]['title'] = object.week_title
-      })
-      cb(null, weeks);
```

Comment:
Map is for returning a new array, but you are using it like a `forEach`. This should be a reduce, as you are trying to turn an array into an object.
```
var result = res.reduce((acc, object) => {
         acc[object.num].mentors.push(object.githubuser);
         acc[object.num]['title'] = object.week_title
       }, weeks)
```